pkg/apis/imageregistry/v1alpha1/types.go

```diff
@@ -36,6 +36,8 @@ type ImageRegistryConfigStorageS3 struct {
 	Region         string `json:"region,omitempty"`
 	RegionEndpoint string `json:"regionEndpoint,omitempty"`
 	Encrypt        bool   `json:"encrypt,omitempty"`
+	AccessKey      string `json:"accesskey,omitempty"`
+	SecretKey      string `json:"secretkey,omitempty"`
```

Comment:
@legionus @bparees are we ok with having secrets in our custom resource?

Reply:
no, they need to be references to a secret.